credoai/lens/__init__.py

```diff
@@ -6,3 +6,4 @@
 
 from .lens import Lens
 from .pipeline_creator import PipelineCreator
+from .lens_validation import *
```

Comment:
Is it necessary to make `lens_validation` functinality avaiable at this level since it is only used by `Lens`? `import *` is generally not recommended.